modules/installation-azure-service-principal.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+[id="installation-azure-service-principal_{context}"]
+= Creating a service principal
+
+Because {product-title} and its installation program must create Microsoft Azure
+resources through Azure Resource Manager, you must create a service principal
+to represent it.
+
+.Prerequisites
+
+* Install or update the link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-yum?view=azure-cli-latest[Azure CLI].
+* Install the `jq` package.
+* Your Azure account has the required roles for the subscription that you use.
+
+.Procedure
+
+. Log into the Azure CLI:
++
+----
+$ az login
+----
++
+Log into Azure in the web console by using your credentials.
+
+. If your Azure account uses subscriptions, ensure that you are using the right one.
```

Comment:
Re: "the right one" - I suspect this means subscription, not account, though you might want to clarify.